region/lowernorfair/west.json

```diff
@@ -164,6 +164,28 @@
                     }},
                     {"heatFrames": 125}
                   ]
+                },
+                {
+                  "name": "Acid Dive",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessLavaDive",
```

Comment:
There would be another strat for having Gravity I guess.

Reply:
Surprisingly, gravity didnt really change the acid frames, and it would still require `canSuitlessLavaDive`, since it is lava and gravity doesnt fully protect you